### ISSUE TITLE
fix: keyboard navigation does not work when there is an item selected in FluentAutoComplete

### ISSUE THREAD

**Issue by MarvinKlein1508** [microsoft/fast-blazor#4231]
# 🐛 Bug Report

Keyboard navigation is broken for the `FluentAutoComplete` component when `Multiple` is set to `false` and there is already an item being selected.

## 💻 Repro or Code Sample
https://www.fluentui-blazor.net/Autocomplete

1. Navigate to the multiple == false demo
2. Select an item (works with keyboard)
3. Click on the control again
4. Try to select an item with keyboard

## 🤔 Expected Behavior

I expect the keyboard to work the second time as well

## 😯 Current Behavior

You cannot select any item the second time with the keyboard

## 🔦 Context

This is necessary for the case of #4228 otherwise the FocusMethod will open a menu which cannot be navigated with keyboard.

**Comment by vnbaaij** [comment]
It does work when you use the keyboard to 'tab' to the AutoComplete. It indeed does not work if you open the AutoComplete with the mouse and then try to select another option by keyboard

**Comment by MarvinKlein1508** [comment]
@vnbaaij and it doesn't work as mentioned when you focus the control via code.

**Comment by MarvinKlein1508** [comment]
Fixed in v5
